src/Entity/Customer.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace PodPoint\Payments\Entity;
```

Comment:
don't mind the naming of `entities` or `models` (plural preferred) but I feel both of these could be misconstrued to be related to a database which is why I didn't add or feel a need for a namespace.

maybe worth having @AndreiaMatiasSilva's second opinion on this?

Reply:
I don't mind either way, they are kind of Value Objects only at the moment, but they don't need to be in the dedicated folder.